Resources/public/views/recommendations.html.twig

```diff
@@ -0,0 +1,31 @@
+<div id="recommendation-{{ recommendationId }}" class="ez-recommendation">
+    {{ 'Loading recommended items...'|trans }}
+</div>
+
+<script id="recommendation-template-{{ recommendationId }}" type="text/x-handlebars-template">
+    {{ template|raw }}
+</script>
+
+<script>
+    (function() {
+        var recoClient{{ recommendationId }} = new EzRecoRestClient({
+            scenarioId: '{{ scenarioId }}',
+            locationId: '{{ locationId }}',
+            limit: '{{ limit }}',
+            restUrl: '/recommendations/fetch' // TODO: dynamically replace with path from routing.yml
```

Comment:
> No, this is a local rest API call (not a YC call)
Oops.

It is good to have ideas, don't take this the wrong way. But as cool as an idea may sound, it is almost always better to keep it in mind, and try to make it part of a feature, or a user benefit, and take the time to do it right when we decide to. And trust me, I'm struggling hard to apply this to myself :D